services/ui-src/src/measures/2023/shared/globalValidations/validateOPMRates/index.ts

```diff
@@ -0,0 +1,34 @@
+import { OtherPerformanceMeasure } from "measures/2023/shared/CommonQuestions/types";
+
+/* Other Performance Measure Rate Description. Check all rate descriptions 
+to make sure there are no identical descriptions */
+
+export const validateOPMRates = (
+  otherPerformanceMeasure: OtherPerformanceMeasure["OtherPerformanceMeasure-Rates"],
+  errorMessage?: string
+) => {
+  const errorArray: FormError[] = [];
+
+  if (otherPerformanceMeasure) {
+    const opm_descriptions = otherPerformanceMeasure.filter(
+      (item: any) => item.description !== ""
```

Comment:
```suggestion
      (item: any) => !!item.description
```
slight code recommendation to check for undefined values as well, i had ran into errors validating before i wrote anything into the text field